### DISCUSSION TITLE
v0.6.2

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#201] [General]
A new version with PlugData, finally!

Thanks to everyone who's helped me with this one, I'm gonna try to name everyone but it's becoming increasingly more difficult. Here we go:

@chee, @JoshuaACNewman, @60-hz, @POLARITY, @Wasted-Audio , MZY-CBX, JaqueArse, @jaffasplaffa, @hellocatfood, @hAK-klang, daniellumertz, DSB, fredeic, @vasilymilovidov, @hatitrain, BonefishSam

And thanks to @kreth608 for adding new documentation to this release.

Highlights:
- UI improvements:
    - New welcome screen when there is no patch open
    - Better tab behaviour, always shows patch name and saves toolbar space
    - Better place in UI for documentation and automation panel selection
    - Some statusbar icons have been moved to the settings popup
- New logo made by @JoshuaACNewman
- Updated to ELSE rc4, with new [datetime] object and more
- Improved colour theming options thanks to @chee
- Hold alt to see object indices
- Encapsulate action: select a group of objects and hit cmd+shift+e to encapsulate into subpatch
- Many, many bugfixes

Other features:
- Destructive options in settings dialog now have a confirmation popup
- Basic support for non-latin characters in messages, comments and objects (better support throughout whole GUI coming soon)
- Now remembers last opened tab in DAW
- Improved DAW audio settings panel
- Remembers your zoom level across restarts
- Added "Move to back" feature
- ELSE objects that work now: [oscope~], [function], [canvas.*], [mtx.ctl]
- Add to path button in Deken
- Shift + drag can now also disconnect box from cords
- Can now show a [garray] and [array define] content in a separate window
- Improved documentation panel icon
- Hitting esc on an empty objects deletes the object
- Object name is now shown in parameters panel
- Comments are now transparent when editing
- Can now double-click toolbar to maximise

Bugfixes:
- Changes to messages can now be undone
- Fixed maximise behaviour on Linux
- Correct behaviour for [key], [keyup] and [keyname]
- Fixed various hradio/vradio bugs
- Fixed crash when dragging number box inside a graph
- Fixed issue where dragging a numbox would change other UI objects
- Fixed object creation offset in Renoise
- Fixed "consistency check failure" bug when graph name is empty
- Fixed crash when creating many segmented connections
- Fixed opening files from OS when there is a space in the name
- Fixed scrolling issue with drawables and mousepad
- Fixed a deadlock in DAW version
- Fixed phaser~ crash
- Greyed out automation add button when locked
- Fixed crashes when dragging parameters below zero in [hradio] and [garray]
- Fixed min/max not working for numboxes
- Added option to use non-native file dialog. Some DAW's, like Ardour, don't like native file dialogs
- Fixed default themes for [vslider] and [hslider] (instead of [vsl]/[hsl])
- Fixed problem with Deken packages not showing up
- Fixed array crashes
- Fixed crash when clicking reconnect button
- Text editors automatically close when clicking inlets/outlets
- Can click on subpathers in temporary lock to open
- Fixed several autocomplete bugs
- Fixed mousepad issues
- Fixed pitchbend issues
- Fixed graph selection problem
- Alignment line for straightened connection cords now appear next to the connection
- Fixed crash when double-clicking the close button
- Fixed initialisation issues on Windows
- Fixed [text define] editor issues on Windows
- Fixed bug in [clone] object
- Fixed maximising behaviour on Linux

Other:
- [cnv] no longer intercepts mouseclicks
- Uses Pd shortcuts by default now, there is an option in settings to revert to Max shortcuts
  Note: If you use and AZERTY keyboard, you have to remap you keyboard shortcuts for them to work, or use Max shortcuts instead.

<hr /><em>This discussion was created from the release <a href='https://github.com/timothyschoen/PlugData/releases/tag/'>v0.6.2</a>.</em>

**Comment by alfonso73** [comment]
Great advancements!
